cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/ClusterInfo/ClusterInfo.tsx

```diff
@@ -0,0 +1,64 @@
+// React imports
+import React from 'react';
+
+// Library imports
+import { CdsButton } from '@cds/react/button';
+import { CdsIcon } from '@cds/react/icon';
+import { CdsDivider } from '@cds/react/divider';
+
+import './ClusterInfo.scss';
+
+function ClusterInfo(props: any) {
+    return (
+        <div className="section-raised" cds-layout="container:lg align:center p:md wrap:none">
+            <div cds-layout="vertical">
+                <div cds-layout="horizontal gap:md align:fill align:vertical-center p-b:md" cds-text="subsection">
+                    <div cds-layout="horizontal">
+                        <div cds-layout="horizontal gap:sm wrap=none align:vertical-center">
+                            <CdsIcon cds-layout="m-r:sm" shape="cluster" size="lg" className="icon-blue"></CdsIcon>
+                            <div cds-text="section">{props.name}</div>
+                        </div>
+                        <CdsDivider orientation="vertical" cds-layout="align:right"></CdsDivider>
+                    </div>
+
+                    <div cds-layout="horizontal">
+                        <div cds-layout="vertical align:left m-r:xs">
+                            <label>Provider</label>
+                            <div cds-layout="horizontal">
+                                <div>{props.provider}</div>
+                            </div>
+                        </div>
+                        <CdsDivider orientation="vertical" cds-layout="align:right"></CdsDivider>
+                    </div>
+                    <div cds-layout="horizontal">
+                        <div cds-layout="vertical m-r:xs">
+                            <label>Status</label>
+                            <div cds-layout="horizontal">
+                                {props.status == 'Running' ? (
+                                    <CdsIcon cds-layout="m-r:sm" shape="connect" size="md" status="success"></CdsIcon>
+                                ) : (
+                                    <CdsIcon cds-layout="m-r:sm" shape="disconnect" size="md" status="danger"></CdsIcon>
+                                )}
+                                <div>{props.status}</div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div cds-layout="vertical gap:md align:vertical-center">
+                    <CdsDivider></CdsDivider>
```

Comment:
If you move this divider outside of this div so it comes right after line 46, does the divider stretch to the full width of the card? That would follow the mockups a little more closely if you are able to make it work.